config.yaml

```diff
@@ -81,7 +81,7 @@ options:
     description: Whether requests should be written to the `rewrite-target`
     type: boolean
   rewrite-target:
-    default: "/"
+    default: ""
```

Comment:
Won't it impact existing deployments ?

Reply:
`rewrite-target` will be `"/"` if everything is default.

```python3
    @property
    def _rewrite_target(self) -> Any:
        """Return the rewrite target from config or relation."""
        return self._get_config_or_relation_data("rewrite-target", "/")

```

Reply:
Should that be noted in the option description?

Reply:
Added, thanks!